client/src/AppBundle/Controller/Admin/FixtureController.php

```diff
@@ -78,4 +80,57 @@ public function completeReportSectionsAction(Request $request, $reportId, Kernel
 
         return new JsonResponse(['Report updated']);
     }
+
+    /**
+     * @Route("/authenticateUser", methods={"GET"})
+     * @Security("has_role('ROLE_ADMIN')")
+     */
+    public function authenticateUser(Request $request, KernelInterface $kernel)
+    {
+        if ($kernel->getEnvironment() === 'prod') {
+            throw $this->createNotFoundException();
+        }
+
+        $email = $request->query->get('email');
+        $password = $request->query->get('password');
+
+        $payload = <<<JSON
+{
+    "email": $email,
+    "password": $password
+}
+JSON;
```

Comment:
Should we use json_encode to be consistent with the `createUser` method?

Reply:
Good point yeah, I'll switch to using that.